src/EPR.Calculator.API/Utils/FinancialYear.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System.Globalization;
+using System.Text.RegularExpressions;
+
+namespace EPR.Calculator.API.Utils
+{
+    public record FinancialYear(DateTime value)
+    {
+        /// <summary>
+        /// Gets the financial year.
+        /// </summary>
+        public DateTime Value { get; } = value;
+
+        /// <summary>
+        /// Converts the financial year to the previous calendar year.
+        /// </summary>
+        /// <returns>The previous calendar year as a <see cref="DateTime"/>.</returns>
+        public DateTime ToCalendarYear() => new(this.value.Year - 1, 1, 1, 0, 0, 0, DateTimeKind.Utc);
```

Comment:
Not sure what this logic is? Can we discuss

Reply:
Why are we using UTC?

Reply:
@udaydefra - This was the fix used for one of the error given by Sonar Qube in the service project and I kept it same in API as well, if you want I can remove from API project?